sage.py

```diff
@@ -84,14 +69,14 @@ def load_IANA_mapping():
             raise RuntimeError('Cannot download IANA ports')
     table = csv.reader(content.splitlines())
 
-    # Drop headers (Service name, port, protocol, description, ...)
-    headers = next(table)
+    # Drop headers (service name, port, protocol, description, ...)
+    next(table)
 
     # Note that ports might have holes
     ports = {}
     for row in table:
         # Drop missing port number, Unassigned and Reserved ports
-        if row[1] and 'Unassigned' not in row[3]:# and 'Reserved' not in row[3]:
+        if row[1] and 'Unassigned' not in row[3]:  # and 'Reserved' not in row[3]:
```

Comment:
The "Reversed" that is commented out, should it be used or can it be removed?

Reply:
Reserved, not reversed. Keep it in, maybe IANA changes its mind again.